include/avif/avif.h

```diff
@@ -1603,6 +1603,14 @@ typedef struct avifEncoder
     // Version 1.2.0 ends here. Add any new members after this line.
     // --------------------------------------------------------------------------------------------
 
+    // creationTime and modificationTime are only used when encoding an image sequence (animated
+    // image).
+    //
+    // creationTime and modificationTime are in seconds since midnight, Jan. 1, 1970 UTC (the Unix
+    // epoch). If set to 0 (the default), libavif uses the current time.
+    uint64_t creationTime;
+    uint64_t modificationTime;
```

Comment:
Please review two API design decisions.

1. Use the Unix epoch (because it is more commonly used) instead of the 1904 epoch used by ISO BMFF.
2. Use the value 0 to indicate "use the current time". This is fine because no AVIF files were created or modified in 1970.